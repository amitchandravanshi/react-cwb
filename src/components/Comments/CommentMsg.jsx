import React from "react";
import Card from "@material-ui/core/Card";
import CardContent from "@material-ui/core/CardContent";
import { withStyles } from "@material-ui/core/styles";
import QuestionAnswer from "@material-ui/icons/QuestionAnswer";
const styles = theme => ({
  card: {
    margin: "1rem 0"
  },
  content: {
    padding: "3rem",
    display: "flex",
    justifyContent: "center",
    alignItems: "center",
    flexWrap: "wrap",
    flexDirection: "column"
  }
});
const CommentMsg = props => {
  const { classes, msg } = props;
  return (
    <Card className={classes.card}>
      <CardContent>
        <div className={classes.content}>
          <QuestionAnswer />
          <span>{msg}</span>
        </div>
      </CardContent>
    </Card>
  );
};
export default withStyles(styles)(CommentMsg);
