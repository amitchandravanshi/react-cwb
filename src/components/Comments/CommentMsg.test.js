import React from "react";
import { shallow, mount } from "enzyme";
import toJson from "enzyme-to-json";
import CommentMsg from "./CommentMsg";
const props = {
  classes: {}
};
describe("<CommentHeader> ", () => {
  it("<CommentHeader> snapshot ", () => {
    const wrapper = mount(<CommentMsg {...props} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
