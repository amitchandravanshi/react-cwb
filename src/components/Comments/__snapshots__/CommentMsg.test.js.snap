// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CommentHeader>  <CommentHeader> snapshot  1`] = `
<CommentMsg
  classes={
    Object {
      "card": "CommentMsg-card-1",
      "content": "CommentMsg-content-2",
    }
  }
/>
`;
